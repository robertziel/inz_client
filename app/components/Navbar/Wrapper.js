import styled from 'styled-components';

const Wrapper = styled.nav`
  background: #fafbfc;
  box-shadow: 0 10px 35px rgba(4, 9, 20, 0.04),
    -18px -30px 23px rgba(4, 9, 20, 0.04), 0 1px 8px rgba(4, 9, 20, 0.06),
    0 2px 2px rgba(4, 9, 20, 0.04);
  height: 60px;

  .navbar-right {
    float: right;
    width: 60px;
    height: 60px;
    margin-right: 10px;
  }
`;

export default Wrapper;
