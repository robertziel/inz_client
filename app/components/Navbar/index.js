import React from 'react';

import LanguageToggle from 'components/LanguageToggle/index';
import SignOutButton from 'components/SignOutButton/index';

import Nav from './Nav';

function Navbar() {
  return (
    <Nav>
      <div className="navbar-right">
        <SignOutButton />
        <LanguageToggle />
      </div>
    </Nav>
  );
}

export default Navbar;
