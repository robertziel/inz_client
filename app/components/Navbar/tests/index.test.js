import React from 'react';
import { mount } from 'enzyme';
import { IntlProvider } from 'react-intl';
import { Provider } from 'react-redux';
import { browserHistory } from 'react-router-dom';
import configureStore from '../../../configureStore';

import Navbar from '../index';

describe('<Navbar />', () => {
  it('should render and match the snapshot', () => {
    const store = configureStore({}, browserHistory);

    const wrapper = mount(
      <IntlProvider locale="en">
        <Provider store={store}>
          <Navbar />
        </Provider>
      </IntlProvider>,
    );
    expect(wrapper.html()).toMatchSnapshot();
  });
});
