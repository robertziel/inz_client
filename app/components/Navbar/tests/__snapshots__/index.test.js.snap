// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> should render and match the snapshot 1`] = `"<nav class=\\"Nav-x8yy12-0 jgOoGS\\"><div class=\\"navbar-right\\"><div class=\\"sign-out Wrapper-vnjem8-0 bcewYQ\\"><span aria-hidden=\\"true\\" class=\\"fa fa-power-off\\"></span></div><div class=\\"Wrapper-sc-1k1j1l-0 YFgOt\\"><select><option value=\\"en\\">English</option><option value=\\"pl\\">Polski</option></select></div></div></nav>"`;
